webapp/components/textbox.jsx

```diff
@@ -176,6 +177,21 @@ export default class Textbox extends React.Component {
     render() {
         const hasText = this.props.value && this.props.value.length > 0;
 
+        let editHeader = (
```

Comment:
Do you mind changing this to
```
let editHeader;
if (this.props.previewMessageLink) {
    // special case
} else {
    // default "Edit Message" case
}
```
? We usually do that when the default case isn't a simple string or primitive.